src/LibraryManager.Vsix/Commands/ManageLibrariesCommand.cs

```diff
@@ -41,45 +41,37 @@ private void BeforeQueryStatus(object sender, EventArgs e)
             button.Enabled = KnownUIContexts.SolutionExistsAndNotBuildingAndNotDebuggingContext.IsActive;
         }
 
-        private void Execute(object sender, EventArgs e)
+        private async void ExecuteAsync(object sender, EventArgs e)
         {
             Telemetry.TrackUserTask("ManageLibraries");
 
-            Project project = VsHelpers.GetProjectOfSelectedItem();
+            Project project = await VsHelpers.GetProjectOfSelectedItemAsync();
 
             if (project != null)
             {
-                string rootFolder = project.GetRootFolder();
+                string rootFolder = await project.GetRootFolderAsync();
 
                 string configFilePath = Path.Combine(rootFolder, Constants.ConfigFileName);
 
                 if (File.Exists(configFilePath))
                 {
-                    VsHelpers.DTE.ItemOperations.OpenFile(configFilePath);
+                    await VsHelpers.OpenFileAsync(configFilePath);
                 }
                 else
                 {
-                    System.Threading.Tasks.Task.Run(async () =>
-                    {
-                        CancellationToken token = CancellationToken.None;
+                    var dependencies = Dependencies.FromConfigFile(configFilePath);
+                    Manifest manifest = await Manifest.FromFileAsync(configFilePath, dependencies, default(CancellationToken));
+                    manifest.DefaultProvider = "cdnjs";
+                    manifest.Version = Manifest.SupportedVersions.Max().ToString();
 
-                    if (!File.Exists(configFilePath))
-                    {
-                        var dependencies = Dependencies.FromConfigFile(configFilePath);
-                        Manifest manifest = await Manifest.FromFileAsync(configFilePath, dependencies, token);
-                        manifest.DefaultProvider = "cdnjs";
-                        manifest.Version = Manifest.SupportedVersions.Max().ToString();
+                        await manifest.SaveAsync(configFilePath, default(CancellationToken));
```

Comment:
Use `CancellationToken.None` instead of `default(CancellationToken)` to indicate that you don't intent to cancel it.

Reply:
(This also applies multiple places)